secrets-gradle-plugin/src/main/java/com/google/android/libraries/mapsplatform/secrets_gradle_plugin/Extensions.kt

```diff
@@ -63,7 +63,7 @@ fun Variant.inject(properties: Properties, ignore: List<String>) {
     }.filter { key ->
         key.isNotEmpty() && !ignoreRegexs.any { it.containsMatchIn(key) }
     }.forEach { key ->
-        val value = properties.getProperty(key)
+        val value = properties.getProperty(key).removeSurrounding("\"")
```

Comment:
Please add a test case for this.

Reply:
Thanks for mentioning the test cases, actually the issue is the existing tests, there are already a few cases that give the value with the double quotes, and also check the quotes in the results.
If we want to add this functionality we can use the existing tests and just expect the double quotes to be removed, if not, then the existing tests will not pass and we can't add this functionality!